assets/theme-css/pst/pydata-sphinx-theme.scss

```diff
@@ -1,3 +1,9 @@
+// Next line added by SPHT, to add missing bootstrap-provided variables
```

Comment:
I've been adding things like this to the bottom of `assets/theme-css/styles.css`.

Reply:
My plan was to factor `#pst-back-to-top` out of `base/_base.scss` and into something like `base/_back_to_top.scss` or something.

Reply:
Why not just keep commenting out `  //  z-index: $zindex-tooltip;` like we are currently doing and then factor out the styling we don't need from `base/_base.scss`?

Reply:
I thought the less editing of the original sources needed, the easier it would be to stay in sync.